### ISSUE TITLE
Generate multiple output formats with one cli call

### ISSUE THREAD

**Issue by albertored** [stoplightio/spectral#1970]
**User story.**
As a spectral user, I would like to be able to generate lint reports in different formats with a single cli call without the need or lining the same spec multiple times (it can be time-consuming for huge specs).

**Describe the solution you'd like**
An option is to change `--output` and `--format` cli flags to accept a comma separated list of values so that a user can decide to generate more than one report and save it into different files. In particular

```bash
# as today, stylish report on stdout 
spectral lint --ruleset ruleset.yml openapi.json

# as today, json report on stdout 
spectral lint --ruleset ruleset.yml -f json openapi.json

# json report on file report.json, stylish report on file report.txt
spectral lint --ruleset ruleset.yml -f json,stylish -o report.json,report.txt openapi.json

# json report on file report.json, stylish report on stdout
spectral lint --ruleset ruleset.yml -f json,stylish -o report.json,stdout openapi.json

# TBD error or only json report on stdout
spectral lint --ruleset ruleset.yml -f json,stylish openapi.json
```

I can create a PR if you think is worth adding the feature

**Comment by Luis-GA** [comment]
I am also interested in this feature. I would do something different in the CLI, probably something like robot framework does. Instead a list of formats and files, that could be confusing, I consider It would be better to set more arguments, indicating the format in the argument name and the file in the value. 

For instance:

`spectral lint --ruleset ruleset.yml --junit report.xml  --stylish report.stylish --html report.html openapi.json`

With this approach, the previous CLI could continue for backwards compatibility.
